src/contracts/ownership/InitializedOwnable.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.7.5;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @dev A contract extending Openzeppelin's Ownable contract that allows to
```

Comment:
uber^2nit: Maybe for the contract, we should add `@title` and `@author` tags as well.